tasks/44_time-simulation/time-simulation.js

```diff
@@ -0,0 +1,20 @@
+export class SimTime {
+  constructor() {
+    this.currentTime = 0;
+    this.speed = 1;
+    this.realTime = 0;
+  }
+
+  get() {
+    return this.currentTime;
+  }
+
+  setSpeed(newSpeed) {
+    this.speed = newSpeed;
+  }
+
+  update(time) {
+    this.currentTime += (time - this.realTime) * this.speed;
```

Comment:
```suggestion
const delta = time - this.realTime
const deltaAccelerated = delta * this.speed
this.currentTime += deltaAccelerated
```
Don't have to put everything in one line.
Sometimes the code is more clear with some additional and "unnecessary" variables. But this variables give something to the clarity of the code.